src/imgix-core-js.js

```diff
@@ -17,25 +17,34 @@
   var VERSION = '2.2.1';
   // regex pattern used to determine if a domain is valid
   var DOMAIN_REGEX = /^(?:[a-z\d\-_]{1,62}\.){0,125}(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)[a-z\d]{1,63}$/i;
+  // minimum generated srcset width
+  var MIN_SRCSET_WIDTH = 100;
+  // maximum generated srcset width
+  var MAX_SRCSET_WIDTH = 8192;
+  // returns an array of width values used during srcset generation
+  var DEFAULT_SRCSET_WIDTHS = _generateTargetWidths(MIN_SRCSET_WIDTH, MAX_SRCSET_WIDTH);
+
   // returns an array of width values used during scrset generation
-  var TARGET_WIDTHS = (function() {
+  function _generateTargetWidths(minWidth, maxWidth) {
     var resolutions = [];
-    var prev = 100;
     var INCREMENT_PERCENTAGE = 8;
-    var MAX_SIZE = 8192;
-  
+    var minSize = minWidth;
+    var maxSize = maxWidth;
```

Comment:
Should these values be coerced into integers with a `Math.floor` at this point? It's a little unexpected but right now it seems like there's nothing stopping me from passing `2000.5` as my max width and really screwing things up.

Reply:
Ah yeah, good catch. I should go back and do the same everywhere else.